web/src/main/java/se/inera/intyg/webcert/web/csintegration/integration/CSIntegrationRequestFactory.java

```diff
@@ -88,6 +89,15 @@ public SaveCertificateRequestDTO saveRequest(Certificate certificate) {
             .build();
     }
 
+    public DeleteCertificateRequestDTO deleteCertificateRequest() {
+        return DeleteCertificateRequestDTO.builder()
```

Comment:
Please add patient before merging.

Reply:
Can't add patient without getting it communicated from CS, breaking api against frontend, or making extra call against getCertificate in CS